# You do not change this file.
# Please go to the rec.py file and finish the class there by following the insturctions there.

In [1]:
import rec


## Below are testing code and expected outputs. 

rec1 = rec.Rectangle(10,5)
print('length:', rec1.length)
print('width:', rec1.width)
print('area:', rec1.area())
print('perimeter', rec1.perimeter())
print('\n')

rec1.modify(15,10)
print('length:', rec1.length)
print('width:', rec1.width)
print('area:', rec1.area())
print('perimeter', rec1.perimeter())


length: 10
width: 5
area: 50
perimeter 30


length: 15
width: 10
area: 150
perimeter 50


# The correct output should be the following: 

    length: 10
    width: 5
    area: 50
    perimeter 30


    length: 15
    width: 10
    area: 150
    perimeter 50